components/automate-cli/pkg/verifyserver/server/api/v1/softwareversion_test.go

```diff
@@ -0,0 +1,185 @@
+			expectedError: nil,
+			query:         "opensearch",
+		},
+		{
+
+			description:  "200:success software version check route when query is postgres",
+			expectedCode: 200,
+			responseBody: models.SoftwareVersionDetails{
+				Passed: true,
+				Checks: []models.Checks{
+					{
+						Title:         "stat availability",
+						Passed:        true,
+						SuccessMsg:    "stat is available",
+						ErrorMsg:      "",
+						ResolutionMsg: "",
+					},
+					{
+						Title:         "mkdir availability",
+						Passed:        true,
+						SuccessMsg:    "mkdir is available",
+						ErrorMsg:      "",
+						ResolutionMsg: "",
+					},
+					{
+						Title:         "Linux Version Check",
+						Passed:        true,
+						SuccessMsg:    "Ubuntu version is 20.04",
+						ErrorMsg:      "",
+						ResolutionMsg: "",
+					},
+				},
+			},
+			expectedBody:  `{"status":"SUCCESS","result":{"passed":true,"checks":[{"title":"stat availability","passed":true,"success_msg":"stat is available","error_msg":"","resolution_msg":""},{"title":"mkdir availability","passed":true,"success_msg":"mkdir is available","error_msg":"","resolution_msg":""},{"title":"Linux Version Check","passed":true,"success_msg":"Ubuntu version is 20.04","error_msg":"","resolution_msg":""}]}}`,
+			expectedError: nil,
+			query:         "postgres",
+		},
+		{
+
+			description:  "200:success software version but OS is not supported",
+			expectedCode: 200,
+			responseBody: models.SoftwareVersionDetails{
+				Passed: true,
+				Checks: []models.Checks{
+					{
+						Title:         "stat availability",
+						Passed:        true,
+						SuccessMsg:    "stat is available",
+						ErrorMsg:      "",
+						ResolutionMsg: "",
+					},
+					{
+						Title:         "mkdir availability",
+						Passed:        true,
+						SuccessMsg:    "mkdir is available",
+						ErrorMsg:      "",
+						ResolutionMsg: "",
+					},
+					{
+						Title:         "Linux Version Check",
+						Passed:        false,
+						SuccessMsg:    "",
+						ErrorMsg:      "Kali Linux version is not supported by automate",
+						ResolutionMsg: "Ensure Kali Linux correct version is installed on the node",
+					},
+				},
+			},
+			expectedBody:  `{"status":"SUCCESS","result":{"passed":true,"checks":[{"title":"stat availability","passed":true,"success_msg":"stat is available","error_msg":"","resolution_msg":""},{"title":"mkdir availability","passed":true,"success_msg":"mkdir is available","error_msg":"","resolution_msg":""},{"title":"Linux Version Check","passed":false,"success_msg":"","error_msg":"Kali Linux version is not supported by automate","resolution_msg":"Ensure Kali Linux correct version is installed on the node"}]}}`,
```

Comment:
same use  &models.Checks{} and get success/ error message from variable(like u have done for **responseBody**) or create global variable for entire expectedBody

Reply:
I have declared the expected body as the const as well as the repeating strings in the const as well